### PR TITLE
Combine WindowsRawInputHandler and WindowsMouseHandler into a single implementation

### PR DESCRIPTION
**Pull request by peppy**
Having it as a separate handler made it a bit of a nightmare to configure things like sensitivity. Since it's intended to be an internal workaround/override, combining with `MouseHandler` itself works better than expected.

- [x] Depends on #4271 mostly for test scene usage.

### CODE REVIEW

File: osu.Framework/Platform/Windows/WindowsMouseHandler.cs [hunk 1]
```diff
@@ -1,30 +1,123 @@
...
+using SDL2;
 
 namespace osu.Framework.Platform.Windows
 {
-    public class WindowsMouseHandler : MouseHandler
+    /// <summary>
+    /// A windows specific mouse input handler which overrides the SDL2 implementation of raw input.
+    /// This is done to better handle quirks of some devices.
+    /// </summary>
+    internal unsafe class WindowsMouseHandler : MouseHandler
```
Comment (bdach) [inline osu.Framework/Platform/Windows/WindowsMouseHandler.cs:19]: If #4272 goes in, this will have `Description == "WindowsMouse"`.

**Comment by bdach** [comment]
Am I reading this right, that windows won't be able to turn *off* raw input now? From an osu!-centric viewpoint I guess that might be fine, as I would hope that nobody was seriously playing osu! with mouse accel on, but it may be more questionable when viewing this issue from a game framework standpoint.

**Comment by peppy** [comment]
turn off relative mode

File: osu.Framework/Platform/Windows/WindowsMouseHandler.cs (hunk 1 above)
Comment (peppy) [reply osu.Framework/Platform/Windows/WindowsMouseHandler.cs:19]: Fixed in #4272.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**